include/seqan3/io/detail/record.hpp

```diff
@@ -137,20 +137,26 @@ auto const & get_or_ignore(record<field_types, field_ids> const & r)
 }
 
 //!\copydoc seqan3::detail::get_or_ignore
-template <size_t i, template <tuple_like ...types_> typename tuple_like_t, typename ...types>
-auto & get_or_ignore(tuple_like_t<types...> & t)
+template <size_t i, typename tuple_t>
+//!\cond
+    requires tuple_like<tuple_t>
+//!\endcond
```

Comment:
The following would be shorter and does not need the `cond` part – is there a reason to not write it this way?
```suggestion
template <size_t i, tuple_like tuple_t>
```